indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -421,6 +421,7 @@ public long push() throws IOException
       case "hdfs":
       case "viewfs":
       case "maprfs":
+      case "s3a":
```

Comment:
I don't think we really want uniformity. There's no good reason for people with S3 deep storage to have to load the druid-hdfs-storage extension or hadoop classes on their historical nodes.

The idea behind how things work now is that even if a hadoop indexing job uses the hadoop fs classes to push data to s3 deep storage, you'd still configure your historicals with s3 deep storage, not hdfs.